### PR TITLE
feat(docs): update breakpoints page to use new layout

### PR DESCRIPTION
**Pull request by chanceaclark**
## What?
Update Breakpoints page to use new layout.

## Screenshots/Screen Recordings
screencapture-localhost-3000-breakpoints-2022-01-14-11_38_36
Screen Shot 2022-01-14 at 11 38 50
Screen Shot 2022-01-14 at 11 39 04

## Testing/Proof

N/A

### CODE REVIEW

File: packages/docs/components/Code/styled.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const StyledCode = styled.code<CodeProps>`
     highlight &&
     !primary &&
     css`
-      background-color: ${theme.colors.warning10};
+      background-color: ${theme.colors.warning20};
```
Comment (chanceaclark) [inline packages/docs/components/Code/styled.tsx:13]: Updated to be just a little bit brighter

File: packages/docs/pages/breakpoints.tsx [hunk 2]
```diff
@@ -3,61 +3,94 @@ import { breakpointValues } from '@bigcommerce/big-design-theme';
...
+        ]}
+      />
+    </Panel>
 
-          return <StyledBox backgroundColor="secondary20" />;
-        }}
-        {/* jsx-to-string:end */}
-      </CodePreview>
+    <Panel header="Do's and Don'ts" headerId="guidelines">
+      <GuidelinesTable recommended={['Use built in responsive props, where applicable.']} discouraged={[]} />
```
Comment (chanceaclark) [inline packages/docs/pages/breakpoints.tsx:93]: Let me know if there might be additions to this.

**Review by golcinho — APPROVED** [review]

**Review by jorgemoya — COMMENTED** [review]
Should we include an example of how to breakpoint inline in props of existing components, like the spacing props? 

Maybe add a Do to use inline breakpoint values when possible (instead of extending)?

**Comment by chanceaclark** [comment]
> Should we include an example of how to breakpoint inline in props of existing components, like the spacing props?

So I thought about this, but we show examples in each of the spacing props on how to use it. My rationale for adding another example is that a user may not see the other example, so probably showing an example here might be beneficial 🤷‍♂️ 

> Maybe add a Do to use inline breakpoint values when possible (instead of extending)?

So I did add `Use built in responsive props, where applicable.`. To me that's pretty similar 🤔

**Comment by jorgemoya** [comment]
> So I did add `Use built in responsive props, where applicable.`. To me that's pretty similar 🤔

Ah yeah, that is good enough. 

I'm conflicted because if I was a user of BigDesign and I wanted to remember the correct syntax for using built in props I would look here first. 🤔

**Comment by chanceaclark** [comment]
I'll go ahead an add an example then!

**Review by jorgemoya — APPROVED** [review]
Perfect

**Comment by chanceaclark** [comment]
Those last two force pushes were to fix some grammatical errors I found.

**Pull request merged**